src/Engine/Util/Input.hpp

```diff
@@ -15,6 +15,11 @@ class InputHandler {
             CONTROL, ///< Control key.
             NEW, ///< Create new hymn.
             OPEN, ///< Open existing hymn.
+            CAMERA, ///< Move camera.
```

Comment:
That, to me, implies that once the ACTIVATECAMERA button is pressed, the camera is now acticated. That's not how it works. The CAMERA button has to be held in order to move the camera.